### ISSUE TITLE
Login Error

### ISSUE THREAD

**Issue by guenslou2001** [JestonBlu/RobinHood#58]
I am having a hard time calling in my account with my username and password through R. 

```
library(RobinHood)
username <- username
password <- password
RobinHood(username, password)
```

I get the following error: "Login not successful, check username and password."
I know for certain the username and password used are correct. I'm not sure if this has to do with Robinhood API or access tokens.

I apologize if this was addressed somewhere else. I did look through all of your opened/closed issues, but couldn't find anything. 

Thanks in advance.

**Comment by JestonBlu** [comment]
It looks like RobinHood changed their authentication. I am following an issue on the python version of this package and it looks like they have created a fix for it. I will start working on implementing it this week.

**Comment by JestonBlu** [comment]
@guenslou2001 I have pushed a fix to this issue and tested it out on most of the functions. Would you mind installing the development version through github and give it a try? 75383582b17cdd75240132afaaa73230c8681d73

**Comment by guenslou2001** [comment]
@JestonBlu Thanks for the update. 
I tried the developer version but I'm now getting the following error

> Error: lexical error: invalid char in json text.
                                       <h1>Server Error (500)</h1>
                     (right here) ------^
>

**Comment by JestonBlu** [comment]
okay, ill look again tonight... probably a typo

**Comment by JestonBlu** [comment]
Im not able to reproduce your error. I was able to install the github version and successfully connect to RobinHood. If you installed from RStudio did you restart your session after installing? Can you also verify that your RobinHood package is 1.0.7?

**Comment by JestonBlu** [comment]
@guenslou2001 have you been able to look at this again? Im will probably push a maintenance fix to CRAN  this weekend if I dont hear anymore.

**Comment by guenslou2001** [comment]
@JestonBlu
Sorry for the late reply. I'm on travel at the moment and it's super hard for me to get access to a computer.
Once I'm able, as recommended, I will try restarting my RStudio session. And feel free to push the update. I won't be back home until the 25th of May. 
Thanks

**Comment by JestonBlu** [comment]
@guenslou2001 Thanks, since it appears to be working on multiple computers I have, I feel pretty good about the issue so I will go ahead and push a maintenance release this weekend. Let me know if you are still having issues when you are back and we can look into it further.

**Comment by guenslou2001** [comment]
@JestonBlu 
I'm back from my vacation and I had the chance to test the package again. I tried 3 different computers, updated my Rstudio and Base R--- nothing. I kept getting the same error. I then noticed that I had a "%" character in my password. I decided to update my password and remove the character in case it was causing issues with JSONLITE. and boom! that was the problem. Now, everything works like a charm. 
So, for people with characters in their username/password, they most likely will get the error I was getting. 
Thanks a lot for taking the time to update the package.
